bin/compile_model.sh

```diff
@@ -310,12 +324,10 @@ if [[ !  -f $EDIR/mysource/ ]] && [[ $ICEFLG -eq 0 ]] ; then
         echo "############# copy from  $EDIR/mysource to build folder"
         rm $targetdir/mod_hycom.F
         rm $targetdir/hycom.F
-        rm $targetdir/mod_cpl_oasis.f90
-        rm $targetdir/mod_cpl_oasis_init.f90
+        rm $targetdir/mod_cpl_oasis.*
         cp $EDIR/mysource/mod_hycom.F $targetdir/
         cp $EDIR/mysource/hycom.F $targetdir/
-        cp $EDIR/mysource/mod_cpl_oasis.f90 $targetdir/
-        cp $EDIR/mysource/mod_cpl_oasis_init.f90 $targetdir/
+        cp $EDIR/mysource/mod_cpl_oasis.f* $targetdir/
```

Comment:
This is for compiling nextsim, it will not work and must be returned to the original.

Reply:
The part for compiling the oasis modules still must be modified, reverted back to the way it was.  I cannot compile hycom-nextsim with the present suggestions for changes.

Reply:
It is used in the develop branch.

Reply:
The current code on the development branch look like this:
`
        rm $targetdir/mod_hycom.F
        rm $targetdir/hycom.F
        rm $targetdir/mod_cpl_oasis.f90
        rm $targetdir/mod_cpl_oasis_init.f90
        cp $EDIR/mysource/mod_hycom.F $targetdir/
        cp $EDIR/mysource/hycom.F $targetdir/
        cp $EDIR/mysource/mod_cpl_oasis.f90 $targetdir/
        cp $EDIR/mysource/mod_cpl_oasis_init.f90 $targetdir/
`

see 
[https://github.com/nansencenter/NERSC-HYCOM-CICE/blob/develop/bin/compile_model.sh](url)

Reply:
Thanks, it is clear how to change it.